shared/js/toc.jsx

```diff
@@ -225,24 +224,32 @@ loop.shared.toc = (function(mozL10n) {
       }
 
       // XXX akita: Code below is just for testing purpose
+      var urlObj;
       var btn = this.refs.addSiteBtn;
       btn.textContent = "Loading...";
       btn.disabled = true;
```

Comment:
I think we can remove all the testing code, added because of the waiting for the metadata